Modules/Filtering/ImageFeature/test/CMakeLists.txt

```diff
@@ -13,6 +13,9 @@ set(ITKImageFeatureTests
     itkBilateralImageFilterTest.cxx
     itkBilateralImageFilterTest2.cxx
     itkBilateralImageFilterTest3.cxx
+    itkFastBilateralImageFilterTest.cxx
+    itkFastBilateralImageFilterTest2.cxx
+    itkFastBilateralImageFilterTest3.cxx
```

Comment:
I'm not a fan of test names like "Test2", "Test3", etc. Hope we can eventually move towards more GTest style test, and use more meaningful names for tests.